suite-native/react-native-graph/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@suite-native/react-native-graph",
+    "version": "0.3.0",
```

Comment:
Set version to 1.0.0 as our others packages

Reply:
version changed in: 730f95286